src/filters/filters.less

```diff
@@ -28,3 +28,70 @@
     }
   }
 }
+
+pf-filter-panel {
+  .dropdown > .dropdown-menu, .input-group-btn > .dropdown-menu {
+    margin-top: 5px;
+  }
+  input {
+    &:-moz-placeholder            { font-style: italic; padding-left: 10px; } // Firefox 4-18
+    &::-moz-placeholder           { font-style: italic; padding-left: 10px; } // Firefox 19+
+    &:-ms-input-placeholder       { font-style: italic; padding-left: 10px; } // Internet Explorer 10+
+    &::-webkit-input-placeholder  { font-style: italic; padding-left: 10px;} // Safari and Chrome
+  }
+  .filter-panel-container {
+    padding: 10px;
+    .keyword-filter {
+      margin-bottom: 10px;
+    }
+    ul {
+      list-style-type: none;
+      padding-left: 16px;
+    }
+    .category {
+      font-size: 15px;
+      font-weight: 400;
+
+      .category-option-label {
+        vertical-align: text-bottom;
+        font-size: 12px;
+      }
+    }
+  }
+  .list-inline > li {
+    padding-right: 0px;
+    @media(max-width: @screen-md-min) {
+      margin-bottom: 3px;
+    }
```

Comment:
Played with the responsiveness a bunch, there are some issues with the patternfly implementation here and the general layout. I don't think adding the 3px bottom margin does enough to fix this to warrant the need to include bootstrap's variables.less file. I worry that adding that could cause issues for applications pulling in the less files to build (as far as path and overrides and such).